interview-study-notes-for-security-engineering.md

```diff
@@ -52,7 +52,7 @@ I recommend doing a lot of reading and whenever you come across a term you are u
 	- Doesn’t show up in http logs. 
 
 - DNS configs
-	- Start of Authority (SOA). 
+	- Start of Authority (SOA). d
```

Comment:
This `d` looks like a typo?

Reply:
yes, it is. Thanks for removing it and merging the request.